packages/zip-it-and-ship-it/src/runtimes/node/bundlers/zisi/tree_files.ts

```diff
@@ -10,7 +10,8 @@ export const getTreeFiles = async function (srcPath: string, stat: Stats): Promi
 
   return await glob(`${srcPath}/**`, {
     ignore: `${srcPath}/**/node_modules/**`,
-    nodir: true,
+    followSymbolicLinks: false,
```

Comment:
adding this here and in `./published.ts` fixes some tests in `tests/main.test.ts` that otherwise hang indefinitely (see other comment)

Reply:
Although even with this change (which I'm not sure is desirable... we do support symlinks I believe) there are three failing (not hanging) tests. To run those locally:
```
npm run build && nx run @netlify/zip-it-and-ship-it:test:ci -- tests/main.test.ts
```